geomstats/geometry/invariant_metric.py

```diff
@@ -543,16 +544,19 @@ def exp(self, tangent_vec, base_point, n_steps=10, step='rk4',
         """
         group = self.group
         basis = self.lie_algebra.orthonormal_basis(self.metric_mat_at_identity)
+        sign = 1. if self.left_or_right == 'left' else 1.
 
         def lie_acceleration(point, vector):
-            velocity = self.group.compose(point, vector)
+            velocity = self.group.tangent_translation_map(
```

Comment:
Docstring for this function? Even if it is a private function, an equation could help the user.